canonical/candid
# Update SSO buttons

## Done
Update the IDP login buttons according to designs

## QA
- Make sure you have all dependencies:
  - `snap install go`
  - `sudo apt install build-essential bzr git`
- Delete the contents of `cmd/candidsrv/config.yaml` and paste in https://pastebin.canonical.com/p/59WVqnRjbT/
- Run the site `make server`
- Go to http://localhost:8081/login
- Check that there is one SSO button that spans the width of the container
- Delete the contents of `cmd/candidsrv/config.yaml` and paste in https://pastebin.canonical.com/p/nDNZk5dkTW/
- Check that there are two SSO buttons and are equal sizes
- Delete the contents of `cmd/candidsrv/config.yaml` and paste in https://pastebin.canonical.com/p/zpFVfQh6PK/
- Check that there are three SSO buttons and fill the whole width in a row
- Delete the contents of `cmd/candidsrv/config.yaml` and paste in https://pastebin.canonical.com/p/sf5VgDZgmn/
- Check that there are four SSO buttons and they wrap after three

## Issue
Fixes https://github.com/canonical-web-and-design/snap-squad/issues/1839

## Screenshots
Mobile:
![mobile](https://user-images.githubusercontent.com/501889/108877691-62ce6f00-75f7-11eb-9a4f-4b928160ce0b.png)

One button:
![one_button](https://user-images.githubusercontent.com/501889/108877731-6c57d700-75f7-11eb-8281-b35ffd73c714.png)

Two buttons:
![two_buttons](https://user-images.githubusercontent.com/501889/108877760-737ee500-75f7-11eb-9acd-f0882162b6e2.png)

Three buttons:
![three_buttons](https://user-images.githubusercontent.com/501889/108877781-78439900-75f7-11eb-8bca-2e804d389c3e.png)

Four buttons:
![four_buttons](https://user-images.githubusercontent.com/501889/108877809-7e397a00-75f7-11eb-9198-05b2df299aea.png)
